xarray/conventions.py

```diff
@@ -191,7 +195,25 @@ def decode_cf_variable(
     if decode_timedelta:
         var = times.CFTimedeltaCoder().decode(var, name=name)
     if decode_times:
-        var = times.CFDatetimeCoder(use_cftime=use_cftime).decode(var, name=name)
+        # remove checks after end of deprecation cycle
+        if not isinstance(decode_times, CFDatetimeCoder):
+            if use_cftime is not None:
+                emit_user_level_warning(
+                    "Usage of 'use_cftime' as kwarg is deprecated. "
+                    "Please initialize it with CFDatetimeCoder and "
+                    "'decode_times' kwarg.",
```

Comment:
```suggestion
                    "'decode_times' kwarg.\n",
                    "Example usage:\n",
                    "    time_coder = xr.coders.CFDatetimeCoder(use_cftime=True)\n",
                    "    ds = xr.open_dataset(decode_times=time_coder)\n",
```

Reply:
@mathause Would that help in the DeprecationWarning?

Reply:
Yes, that's good.